### PR TITLE
[SYCLomatic #1160] Test for path to helper function dir

### PR DESCRIPTION
**Pull request by daiyaan-ahmed6**
Adds test for printing path to installed helper function directory

Signed-off-by: Ahmed, Daiyaan <daiyaan.ahmed@intel.com>

### CODE REVIEW

File: behavior_tests/src/bt-helper-function-dir/do_test.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+def setup_test():
+    change_dir(test_config.current_test)
+    return True
+
+def migrate_test():
+
+    call_subprocess(test_config.CT_TOOL +" --helper-function-dir")
+    helper_function_dir_root = os.path.abspath(os.path.join(get_ct_path(), os.pardir , os.pardir , "include"))
+    print("Helper functions directory: ", helper_function_dir_root)
+    return is_sub_string(helper_function_dir_root, test_config.command_output)
```
Comment (tomflinda) [inline behavior_tests/src/bt-helper-function-dir/do_test.py:27]: Could you explain why that "if helper_function_dir_root is a substring of test_config.command_out, helper_function_dir_root is an expected output directory?"

Comment (daiyaan-ahmed6) [reply behavior_tests/src/bt-helper-function-dir/do_test.py:27]: The `test_config.command_output` contains the output of `dpct --helper-function-dir` as well as some newlines appended extra to it. To test whether command output is helper function directory, I am checking whether it is a full substring in the command output.

I followed the examples of: 1) bt-version: https://github.com/oneapi-src/SYCLomatic-test/blob/SYCLomatic/behavior_tests/src/bt-version/do_test.py#L26

and 2) bt-help: https://github.com/oneapi-src/SYCLomatic-test/blob/SYCLomatic/behavior_tests/src/bt-help/do_test.py#L23

**Review by tomflinda — APPROVED** [review]
LGTM

**Review by intwanghao — APPROVED** [review]
LGTM

File: behavior_tests/src/bt-helper-function-dir/do_test.py (hunk 1 above)
Comment (zhimingwang36) [reply behavior_tests/src/bt-helper-function-dir/do_test.py:27]: could you show a example value of:  helper_function_dir_root and  test_config.command_output?

**Pull request merged**